internals/cli/cmd_run.go

```diff
@@ -31,16 +31,16 @@ import (
 	"github.com/canonical/pebble/internals/systemd"
 )
 
-const cmdRunSummary = "Run the pebble environment"
+const cmdRunSummary = "Run the {{.DisplayName}} environment"
 const cmdRunDescription = `
 The run command starts pebble and runs the configured environment.
```

Comment:
Need to override this instance too? Probably with `{{.DisplayName}}`.

Reply:
Looks like this comment was resolved but not done? The `cmdRunDescription` still says "The run command starts pebble..."